ui/app/src/components/AceEditor.tsx

```diff
@@ -263,9 +263,9 @@ export default React.forwardRef(function AceEditor(props: AceEditorProps, ref) {
     if (refs.current.pre) {
       refs.current.pre.className = `${[...refs.current.pre.classList]
         .filter((value) => !/craftercms-|makeStyles-/.test(value))
-        .join(' ')} ${classes.editorRoot}`;
+        .join(' ')} ${classes?.editorRoot} ${props.classes?.editorRoot}`;
     }
-  }, [classes.editorRoot]);
+  }, [classes.editorRoot, props.classes]);
```

Comment:
Add const for storing value of props.classes.editorRoot and use that on the effect